pkgs/development/libraries/libucl/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
+, autoreconfHook
+, curl
+, lua
+, openssl
+, enableUrls ? false
```

Comment:
Why disable by default?

Reply:
I'm reflecting how this is set when you build from source. I'm not familiar with this lib, so I'm assuming all the non default options are extras.

Reply:
Could it be they are disabled if the necessary deps are not found? It's a common practice to make the build not fail but disable optional features if they are not explicitly enabled.

Reply:
Maybe yeah. I'm really unsure on what approach to take here. On the other end of the spectrum, enabling everything, seems a bit overkill too.